labs/src/main/scala/org/scalalabs/basic/lab02/ListManipulationExercise01.scala

```diff
@@ -47,7 +46,7 @@ object ListManipulationExercise01 {
    *  - ... etc
    */
   def nthElementInList[T](n: Int, l: List[T]): T = {
-    error("fix me")
+    l.slice(n, n+1).head
   }
```

Comment:
Oh, yeah, I did this before I learned what the apply function did :stuck_out_tongue: